doc/sphinx/user/methods/stokes-solver/solver.md

```diff
@@ -0,0 +1,236 @@
+# Stokes solver
+
+
+
+Problems in ASPECT require solving the Stokes
+problem below for velocity $u$ and pressure
+$p$
+
+```{math}
+:label: eq:strong-form
+-\nabla \cdot (2 \eta(x)\varepsilon(u))+\nabla p &= f \quad \text{ in } \Omega \\
+-\nabla \cdot u&=0 \quad\text{ in } \Omega
```

Comment:
here you only discuss the incompressible Stokes equations, but ASPECT also has compressible solvers. In terms of how you write the equations and what you want to discuss the difference is probably limited to the definition of `\varepsilon(u)` and replacing the `0` with some `g` in the mass conservation equation. I think it would make sense to either say you limit yourself to incompressibility, or modify the equations.